File: android/app/src/main/kotlin/io/github/dey410/gardendlessloader/game/GameMenuController.kt
package io.github.dey410.gardendlessloader.game

import android.webkit.WebView

class GameMenuController(private val webView: WebView) {
    fun open() {
        webView.evaluateJavascript(
            "window.__gardendlessMenu && window.__gardendlessMenu.open()",
            null,
        )
    }
}

File: ios/Runner/GameMenuController.swift
import WebKit

final class GameMenuController {
  private weak var webView: WKWebView?

  init(webView: WKWebView) {
    self.webView = webView
  }

  func open() {
    webView?.evaluateJavaScript("window.__gardendlessMenu && window.__gardendlessMenu.open()")
  }
}
